.eslintrc.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
- [ ] It would be better if our README file is descriptive of the current project. Kindly replace the needed fields with the right information. For example; `To Do List` instead of `Project Name`, etc

Reply:
Done

Reply:
Well resolved 👍🏽